reana/cli.py

```diff
@@ -1574,3 +1574,14 @@ def kubectl_delete_pod(component):  # noqa: D301
                 cmd = 'kubectl delete pod --wait=false -l app={0}'.format(
                     COMPONENT_PODS[component])
                 run_command(cmd, component)
+
+
+@cli.command(name='git-install-eggs')
+def git_install_eggs():
+    """Create eggs-info/ in all REANA infrastructure and runtime components."""
+    reana_ui_index = REPO_LIST_CLUSTER.index('reana-ui')
+    python_cluster_components = (REPO_LIST_CLUSTER[0:reana_ui_index] +
```

Comment:
What about the following and you can get rid of `reana_ui_index`?
```python
python_cluster_components = [c for c in REPO_LIST_CLUSTER if c != 'reana-ui']
```

Reply:
I was unconsciously trying to avoid iterating, but it is such a small list that this way pays off because of readability. Thanks for the input :)

Reply:
Yep, that was the main reason, otherwise it was a bit cryptic.